projects/plugins/jetpack/extensions/blocks/recipe/class-jetpack-recipe-block.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/**
+ * Jetpack Recipe Block
+ *
+ * @package automattic/jetpack
+ */
+
+namespace Automattic\Jetpack\Extensions\Recipe;
+
+use Jetpack_Gutenberg;
+
+/**
+ * Jetpack Recipe Block class.
+ *
+ * Helper class that lets us add schema attributes dynamically because they are not something that is store with the content.
+ * Due to the limitations of wp_kses.
+ *
+ * @since $$next-version$$
+ */
+class Jetpack_Recipe_Block {
+	/**
+	 * Adds recipe schema attributes.
+	 *
+	 * @param array  $attr    Array containing the recipe block attributes.
+	 * @param string $content String containing the recipe block content.
+	 *
+	 * @return string
+	 */
+	public static function render( $attr, $content ) {
+		Jetpack_Gutenberg::load_styles_as_required( 'recipe' );
+
+		wp_enqueue_script(
+			'recipe-print',
+			plugins_url( 'recipe-print.js', __FILE__ ),
+			array(),
+			JETPACK__VERSION,
+			true
+		);
```

Comment:
If we move the recipe js to `view.js`, we'll need to change this to `load_assets_as_required`. We'll also be able to remove the enqueue below.

```suggestion
		Jetpack_Gutenberg::load_assets_as_required( 'recipe' );
```

Reply:
Absolutely, removing the need for an extra file to be loaded makes total sense.

I had to leverage `load_assets_as_required` in favor of `load_styles_as_required` to get the view.js file functions working properly.

This should be taken care of!